### PR TITLE
Added support for Accounts (CS2)

### PR DESCRIPTION
**Pull request by Armando-Ibarra**

### CODE REVIEW

File: lib/checkout_sdk/accounts/accounts_client.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+      ACCOUNTS = 'accounts'
+      ENTITIES = 'entities'
+      INSTRUMENT = 'instruments'
+      PAYOUT_SCHEDULE = 'payout-schedules'
+      FILES = 'files'
+      private_constant :ACCOUNTS, :ENTITIES, :INSTRUMENT, :PAYOUT_SCHEDULE, :FILES
+
+      # @param [ApiClient] api_client
+      # # @param [ApiClient] files_client
```
Comment (martinseco) [inline lib/checkout_sdk/accounts/accounts_client.rb:16]: Remove one of the # from the comment

File: lib/checkout_sdk/accounts/schedule_request.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+module CheckoutSdk
+  module Accounts
+    # @!attribute frequency
+    #   @return [String] {ScheduleFrequencyType}
+    class ScheduleRequest
+      attr_reader :frequency
+
+      protected
+
+      # @param [String] frequency {ScheduleFrequencyType}
```
Comment (martinseco) [inline lib/checkout_sdk/accounts/schedule_request.rb:12]: Add @abstract annotation

**Review by martinseco — APPROVED** [review]
LGTM! Just two comments about docs annotations

File: lib/checkout_sdk/accounts/accounts_client.rb (hunk 1 above)
Comment (Armando-Ibarra) [reply lib/checkout_sdk/accounts/accounts_client.rb:16]: Nice catch

**Pull request merged**